.github/workflows/systest.yml

```diff
@@ -130,13 +130,13 @@ jobs:
           go-version-file: "go.mod"
 
       - name: Run tests
-        timeout-minutes: 60
+        timeout-minutes: 75
```

Comment:
Please revert this - system tests take ~ 42-45 minutes to complete

if certain things go wrong during a test the test will run for 1 hour before timing out - this will just increase the time until we see that something is broken and not improve the stability of the tests 😕

Reply:
Reduced back to 60